azurerm/internal/services/policy/policy_assignment_resource.go

```diff
@@ -113,6 +113,16 @@ func resourceArmPolicyAssignment() *schema.Resource {
 				DiffSuppressFunc: structure.SuppressJsonDiff,
 			},
 
+			"enforcement_mode": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
While testing locally, I was able to update this attribute without recreating the resource. So we should be able to remove this

Reply:
you are right. I removed ForceNew